ChunkArray/chunkarray.js

```diff
@@ -0,0 +1,26 @@
+var chunk = function (arr, size) {
+  let start = 0;
+  let ans = [];
+
+  if (size === 0) {
+    return arr;
+  }
+  for (let i = 0; i < Math.ceil(arr.length / size); i++) {
+    
+    start + size > arr.length
+      ? (ans = [...ans, arr.slice(start, arr.length)])
+      : (ans = [...ans, arr.slice(start, size + start)]);
```

Comment:
```suggestion
  const sliceEndIndex =  start + size > arr.length ? arr.length : size + start
  ans = [...ans, arr.slice(start, sliceEndIndex)]
```
always aim for more code readability. check out this approach, it would be more readable.